site/en/blog/chromium-chronicle-20/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "The Chromium Chronicle #20: Benchmarking Test Harnesses"
+description: >
+  Adding benchmarks is an easy way to prevent performance regressions and
+  help improve performance over time.
+layout: 'layouts/blog-post.njk'
+date: 2021-04-30
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 20:** by John Chen in Bellevue, WA (April, 2021)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+Speed is one of Chrome's four core principles. **Adding benchmarks is an
+easy way to prevent performance regressions and help improve performance
+over time.** Good Benchmarks have a fast iteration cycle, can catch
+performance regressions much earlier than UMA, and are great for measuring
+new feature's performance.
+
+**Benchmarks run regularly in the lab.** When a regression is found, bisect
```

Comment:
'Bisect' is a command or system? This should either be:

* 'the Bisect command automatically finds...' (or something similar)

or

* 'Bisecting automatically finds...'

Reply:
Updated with link to bisect